
void foo_test(void)
{
    do error;
}